service/src/main/java/com/epam/rft/atsy/service/domain/CandidateDTO.java

```diff
@@ -26,7 +26,7 @@
 
   @NotNull
   @Pattern(regexp = "[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}", message = "candidate.error.email.incorrect")
-  @Size(min = 1, max = 400, message = "candidate.error.email.long")
+  @Size(max = 400, message = "candidate.error.email.long")
```

Comment:
This line conflicts with the web-controller-unit-tests branch.

Please fix and also fix the email too long test case.